src/apis/user/user.service.ts

```diff
@@ -85,23 +86,32 @@ export class UserService {
     const checkNickName = await this.userRepository.findOne({
       where: { nickname },
     });
-    const checkPhone = await this.userRepository.findOne({
-      where: { phoneNumber },
-    });
+    // const checkPhone = await this.userRepository.findOne({
+    //   where: { phoneNumber },
+    // });
     const checkEmail = await this.userRepository.findOne({
       where: { email },
     });
     if (checkNickName) {
-      throw new NotFoundException('이미 사용 중인 닉네임 입니다.');
+      throw new ConflictException('중복된 닉네임이 존재합니다.');
     } else if (checkEmail)
-      throw new NotFoundException('이미 사용 중인 이메일 입니다.');
+      throw new ConflictException('중복된 이메일 계정이 존재합니다.');
 
-    if (checkPhone) {
-      throw new NotFoundException('이미 사용 중인 휴대폰 번호입니다.');
-    }
+    // if (checkPhone) {
+    //   throw new NotFoundException('이미 사용 중인 휴대폰 번호입니다.');
+    // }
 
+    //이메일 인증이 안되었을 때
     if (isValid !== true || !isValid)
-      throw new BadRequestException('인증이 완료되지 않았습니다.');
+      throw new BadRequestException('인증이 완료되지 않은 이메일입니다.');
+
+    //비밀번호 형식
+    const passwordRule =
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,16}$/;
+
+    if (passwordRule.test(createUserInput.password) !== true) {
+      throw new NotFoundException('비밀번호 형식이 올바르지 않습니다.');
```

Comment:
```suggestion
      throw new BadRequestException('비밀번호 형식이 올바르지 않습니다.');
```
request 로 들어온 값이 유효하지 않을때는 400번을 response 하는게 일반적입니다.